packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -416,7 +419,22 @@ module.exports = function (log, config) {
     );
   };
 
-  Mailer.prototype.localize = function (message) {
+  Mailer.prototype.localize = async function (message) {
+    if (newEmailPattern.test(message.template)) {
```

Comment:
I think Vijay's PR for feature flagging is already going to handle determining if an email should use the new or old template, so this shouldn't be necessary. Base on [this line](https://github.com/mozilla/fxa/pull/9808/files#diff-8f8d686a1e503181760ab061f610c01e8c357bd0da6620272cf9593d9a02e146R433) you should be able to (once that PR is merged) do something like:

```js
if (featureFlags.isMjmlEnabledForUser(message.email, message.template)) {
  // Needs to return `html` and `text` properties
  const localized = await this.fluentLocalizer.localizeEmail(
    message.template,
    'hello-world'
  );
  localizedEmailHtml = localized.html;
  localizedEmailText = localized.text;
}
```

And I _think_ this should allow you to just use the existing `cadReminderEmail` method and not need a new `cadReminderMJMLEmail`?

You could also define the subject Fluent key when calling `this.send` in the individual template mailer methods. Something like: `this.send({ mailSubject: 'hello-world' })`.

Finally, I see `localizeEmail` is importing and using `context` from `renderer.ts`, but I think you'll need to pass variables down to `this.fluentLocalizer.localizeEmail` instead. I think `message.templateValues` is what you'd pass down.

So... altogether, something like:

```js
if (featureFlags.isMjmlEnabledForUser(message.email, message.template)) {
  const { template, mailSubject, templateVariables } = message;
  const localized = await this.fluentLocalizer.localizeEmail(
    template,
    mailSubject,
    templateVariables,
  );
  localizedEmailHtml = localized.html;
  localizedEmailText = localized.text;
} else {
  // the old way
}
```